### PR TITLE
fixed wrong tag in config segment of fluentd http values.yml

### PR DESCRIPTION
**Pull request by talknopf**

### CODE REVIEW

**Review by juan-coralogix — DISMISSED** [review]

**Comment by amit-mazor** [comment]
@talknopf Please add a description , whats the bug and how do we fix it here

**Comment by amit-mazor** [comment]
@talknopf Please add a description , whats the bug and how do we fix it here

File: fluentd/http/values.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ fluentd:
 
   image:
     repository: coralogixrepo/coralogix-fluentd-multiarch
-    tag: v0.0.5
+    tag: v0.0.6
```
Comment (amit-mazor) [inline fluentd/http/values.yaml:6]: the new image [0.0.6] includes only a prometheus plugin  in a newer version, but it seems like Fluentd doesnt let us override the previous version, meaning this tag doesn't actually change anything currently so no need to update it here. 

If you want to change it, then we need to update he CHANGELOG

Comment (amit-mazor) [inline fluentd/http/values.yaml:6]: the new image [0.0.6] includes only a prometheus plugin  in a newer version, but it seems like Fluentd doesnt let us override the previous version, meaning this tag doesn't actually change anything currently so no need to update it here. 

If you want to change it, then we need to update he CHANGELOG

File: fluentd/http/values.yaml [hunk 2]
```diff
@@ -174,7 +174,6 @@ fluentd:
           applicationName ${record.dig("kubernetes", "#{ENV['APP_NAME']}")}
           subsystemName ${record.dig("kubernetes", "#{ENV['SUB_SYSTEM']}")}
           computerName ${record.dig("kubernetes", "host")}
-          timestamp ${time.strftime('%s%L')}
```
Comment (amit-mazor) [inline fluentd/http/values.yaml:177]: why do we need to delete it ?

Comment (amit-mazor) [inline fluentd/http/values.yaml:177]: why do we need to delete it ?

**Comment by talknopf** [comment]
> @talknopf Please add a description , whats the bug and how do we fix it here

we are marking the logs with "container" tag but the kebernetes plugin tage is filtering with *.kubernetes.** tag.
so i fixed the filter

File: fluentd/http/values.yaml [hunk 3]
```diff
@@ -164,7 +164,7 @@ fluentd:
         @type null
       </match>
 
-      <filter kubernetes.**>
```
Comment (talknopf) [inline fluentd/http/values.yaml:167]: @amit-mazor this is where we fixed the bug

File: fluentd/http/values.yaml (hunk 2 above)
Comment (amit-mazor) [reply fluentd/http/values.yaml:177]: we were always using the timestamp field , its was always a part of the records being sent no ? Im not sure I understand

**Comment by amit-mazor** [comment]
@talknopf I think we should still use the older image tag, 0.0.5 and not 0.0.6 for now, its not giving us anything new currently

**Comment by talknopf** [comment]
@amit-mazor @avivgold098 @oded-dd 
Can we continue this one?

**Review by avivgold098 — APPROVED** [review]

**Review by amit-mazor — APPROVED** [review]

**Pull request merged**